packages/rector-generator/src/ValueObject/Configuration.php

```diff
@@ -208,4 +208,16 @@ private function setName(string $name): void
 
         $this->name = $name;
     }
+
+    private function setNodeTypes(array $nodeTypes): void
+    {
+        foreach ($nodeTypes as $nodeType) {
+            if (! class_exists($nodeType)) {
+                $message = sprintf('Node type "%s" doesn\'t exist or not imported in Rector recipe', $nodeType);
```

Comment:
```suggestion
                $message = sprintf('Node type "%s" does not exist or not imported in Rector recipe', $nodeType);
```

Escaping quotes in strings is very hard to read, error prone and not neccessary